tasks/parse-pdr/index.js

```diff
@@ -36,6 +37,17 @@ function parsePdr(event) {
   return parse.ingest()
     .then((payload) => {
       if (parse.connected) parse.end();
+      // opportunity to filter the granules of interest based on regex in granuleIdExtraction
+      if (config && config.collection && config.collection.granuleIdExtraction) {
```

Comment:
Because this is going to involve updating the config schema, the changelog will need to be updated as well.